resources/patches/ruby_241/libyaml_cve-2014-9130.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/ext/psych/yaml/scanner.c b/ext/psych/yaml/scanner.c
+index af05766..965bdeb 100644
+--- a/ext/psych/yaml/scanner.c
++++ b/ext/psych/yaml/scanner.c
+@@ -1106,13 +1106,6 @@ yaml_parser_save_simple_key(yaml_parser_t *parser)
+             && parser->indent == (ptrdiff_t)parser->mark.column);
+ 
+     /*
+-     * A simple key is required only when it is the first token in the current
+-     * line.  Therefore it is always allowed.  But we add a check anyway.
+-     */
+-
+-    assert(parser->simple_key_allowed || !required);    /* Impossible. */
+-
+-    /*
```

Comment:
Does this patch still apply? It doesn't seem like it should: https://github.com/ruby/ruby/blob/trunk/ext/psych/yaml/scanner.c#L1105-L1126